phpBB/includes/functions.php

```diff
@@ -4317,19 +4296,6 @@ function exit_handler()
 	exit;
 }
 
-/**
-* Casts a numeric string $input to an appropriate numeric type (i.e. integer or float)
-*
-* @param string $input		A numeric string.
-*
-* @return int|float			Integer $input if $input fits integer,
-*							float $input otherwise.
-*/
-function phpbb_to_numeric($input)
```

Comment:
Still being used here: https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/php/ini.php#L138

Reply:
I move that function to functions_compatibility because that class is deprecated